ani-cli

```diff
@@ -228,11 +228,11 @@ get_video_quality_m3u8() {
 		1080)
 			idx=5 ;;
 		best|*)
-			idx="$links_count" ;;
+			idx="auto" ;;
 	esac
 	if [ "$idx" -lt 1 ] || [ "$idx" -gt "$links_count" ];then
 		err "Current video quality is not available (defaulting to best quality)"		 
-		idx="$links_count"
+		idx="auto"
```

Comment:
Is this the thing that fixes the mpv audio only?

Reply:
ye

Reply:
Wasn't this the only use of `links_count`? If yes, we should remove it's assignment as well

Reply:
We require that links count for if condition checking as if user gives out of range index then it will fall back to default, so...
However we can reduce it by merging both m3u8 links and links count variable